defaultModel/src/main/scala/pl/touk/nussknacker/defaultmodel/migrations/ParametersMigration.scala

```diff
@@ -0,0 +1,30 @@
+package pl.touk.nussknacker.defaultmodel.migrations
+
+import pl.touk.nussknacker.engine.api.MetaData
+import pl.touk.nussknacker.engine.graph.evaluatedparam.Parameter
+import pl.touk.nussknacker.engine.graph.node
+import pl.touk.nussknacker.engine.graph.node.{CustomNode, Enricher, Join, Processor, Sink, Source, WithParameters}
+
+class ParametersMigration(
+    private val predicate: List[Parameter] => Boolean,
+    private val mapping: Parameter => Parameter
+) {
+
+  def migrateNode(metaData: MetaData): PartialFunction[node.NodeData, node.NodeData] = {
+    case node: WithParameters if predicate(node.parameters) =>
```

Comment:
Here is a bug. You missed checking of `branchParameters`. WDYT about such approach:
- we replace `node match {}` witch:
```
case n: CustomNode if n.parameter.exists(mappingPF.isDefinedAt) => ... 
...
```
- we join mapping and preicate to `PartialFunction`

Thanks to that, we will be sure that this code won't desynchronize nad code of `predicate` and `mapping`  won't desynchronize as well.

Reply:
Done